src/net/local_ack.rs

```diff
@@ -9,7 +8,7 @@ use std::collections::HashMap;
 #[derive(Debug, Default)]
 pub struct LocalAckRecord {
     // packets waiting for acknowledgement.
-    packets: HashMap<u16, Packet>,
+    packets: HashMap<u16, Box<[u8]>>,
```

Comment:
Don't know if we want a box here or just a slice. ill have to read up on it.

Reply:
I also replied to fletcher: _I changed from `Packet` to box of bytes because packet also contained information not needed to be stored. When you request dropped packets we already know from which `addr` it came and we also don't need to know `delivery type` because we only do something with dropped packets if the packet type is `Reliable`_